utils/update-checkout

```diff
@@ -217,17 +229,30 @@ By default, updates your checkouts of Swift, SourceKit, LLDB, and SwiftPM.""")
         "--config",
         default=os.path.join(SCRIPT_DIR, "update-checkout-config.json"),
         help="Configuration file to use")
+    parser.add_argument(
+        "--github-comment",
+        help='Use github comment to parse pull request for other repository.',
```

Comment:
"Check out related pull requests referenced in the given free-form GitHub-style comment." ?